src/AbstractEnum.php

```diff
@@ -46,6 +46,16 @@ public function __construct($value)
     return array_values(static::_valueCache());
   }
 
+  public static function getConstants()
+  {
+    return array_keys(static::_valueCache());
+  }
+
+  public static function getConstantsAndValues()
```

Comment:
```suggestion
  public static function getConstants()
```
Call this `getConstants` and drop the other new method currently using that name (Line 49).
If you need the keys only you can call `array_keys` downstream